src/views/OnBoardingSubProduct/OnBoardingSubProduct.tsx

```diff
@@ -197,7 +197,7 @@ function OnBoardingSubProduct() {
             </Trans>
           ),
           product: subProduct,
-          forward: (_: any, party: Party) => forwardWithInstitution(party, false),
+          forward: (_: any, party: SelfcareParty) => forwardWithInstitution(party, false),
```

Comment:
Party is the right model to use here
```suggestion
          forward: (_: any, party: Party) => forwardWithInstitution(party, false),
```